src/state/openapi.ts

```diff
@@ -90,6 +99,10 @@ class OpenAPI {
     return resources;
   }
 
+  parentResources(): ResourceSchema[] {
+    return this.resources().filter((resource) => resource.parents().length == 0);
```

Comment:
```suggestion
    return this.resources().filter((resource) => resource.parents().length != 0);
```

Shouldn't this be values of non-zero length?

Reply:
ah ok, by "parent" are you referring to resources that have no parents? I'd argue that "root" resources are are a better way to describe that.

A parent is a specific relationship to other resources, it's a little confusing to use that in the context of "resources without a parent".